crates/katana/rpc/src/katana.rs

```diff
@@ -57,6 +59,17 @@ where
         Ok(self.sequencer.backend().accounts.clone())
     }
 
+    async fn get_storage_multiple(
```

Comment:
we should mimic `getStorageAt` where we can request specific `blockId`.

Reply:
Aye, sure.
I had it in the beginning but it was a bit weird in backend with block exists and state by block id etc Katana sequencer methods unavailable. So I skipped on that 😂 
But let's bring it back!

Reply:
yeah tbh we'll probably remove `KatanaSequencer` in the future.. the trait itself isn't very meaningful imo. Might be better to just pass `Backend` etc to both `StarknetApi` and `KatanaApi` instead.